package.json

```diff
@@ -19,7 +19,7 @@
     "loader-runner": "^2.3.0",
     "loader-utils": "^1.1.0",
     "memory-fs": "~0.4.1",
-    "micromatch": "^3.1.8",
+    "micromatch": "^4.0.2",
```

Comment:
`micromatch` require node@8, it is breaking change

Reply:
Alright, I understand.